### ISSUE TITLE
"Umbraco" should not be lowercased

### ISSUE THREAD

**Issue by iOvergaard** [umbraco/Umbraco-CMS#12770]
In the text "Anonymized site ID, umbraco version, and packages installed." it should be "Umbraco".

Please check that Umbraco is written with a capital U throughout all the translation files and keys.

_Originally posted by @bjarnef in https://github.com/umbraco/Umbraco-CMS/issues/12739#issuecomment-1202370184_

**Comment by bjarnef** [comment]
Another minor thing is that we have this text:

```
<i>We might change what we send on the Detailed level in the future. If so, it will be listed above.
<br>By choosing "Detailed" you agree to current and future anonymized information being collected.</i>
```

The `<i>` has no semantic meaning so general I think it would be better to use `<em>` and `<strong>` in favor of `<i>` and `<b>`. The `<i>` element has historically been used for font icons (probably because of the `i`).

**Comment by MayurDeore** [comment]
I'm working on this.

**Comment by bjarnef** [comment]
@MayurDeore that changes hasn't been merged into `v10/contrib` branch yet, so maybe it should be merged first to adjust this.

**Comment by MayurDeore** [comment]
@bjarnef yes I hadn`t checked that before, thanks for letting me know.

**Comment by iOvergaard** [comment]
@MayurDeore > Thank you for your interest. The relevant code has now been merged with the main branch making this issue relevant to fix. Are you still up for it?

**Comment by bjarnef** [comment]
@iOvergaard it still seem the recent changes in `v10/dev` not has been merged to `v10/contrib` branch, e.g. changes in "detailedLevelDescription" from your PR.

https://github.com/umbraco/Umbraco-CMS/blob/v10/dev/src/Umbraco.Core/EmbeddedResources/Lang/en_us.xml

https://github.com/umbraco/Umbraco-CMS/blob/v10/contrib/src/Umbraco.Core/EmbeddedResources/Lang/en_us.xml

**Comment by MayurDeore** [comment]
@iOvergaard  Yes, I'm eager to get started. But not everything has merged yet, according to @bjarnef comment.

**Comment by nul800sebastiaan** [comment]
I've just done the merge now! 👍

**Comment by MayurDeore** [comment]
Great, I'm going to start working on this.

**Comment by MayurDeore** [comment]
@bjarnef Can I also modify the other translation files that contain the `<i>` and `<b>` tags?


> Another minor thing is that we have this text:
> 
> ```
> <i>We might change what we send on the Detailed level in the future. If so, it will be listed above.
> <br>By choosing "Detailed" you agree to current and future anonymized information being collected.</i>
> ```
> 
> The `<i>` has no semantic meaning so general I think it would be better to use `\<em\>` and `\<strong\>` in favor of `<i>` and `<b>`. The `<i>` element has historically been used for font icons (probably because of the `i`).
> 
> so I would change that text to this instead:
> 
> ```
> <em>We might change what we send on the Detailed level in the future. If so, it will be listed above.
> <br>By choosing "Detailed" you agree to current and future anonymized information being collected.</em>
> ```

**Comment by bjarnef** [comment]
@MayurDeore yes, I think that would be fine.
